oak_runtime/src/node/http/util.rs

```diff
@@ -0,0 +1,151 @@
+//
```

Comment:
Moved common structs from `server.rs` to this submodule. These structs are also used in the tests for the `HttpClientNode`.